doc/User_profile.json

```diff
@@ -79,56 +79,35 @@
 	],
 	"picture": "https://s.gravatar.com/avatar/...png",
 	"shirtSize": "L",
-	"membership": {
-		"Workday": [
+	"groups": [
```

Comment:
Is there a reason behind the difference in the format of ``groups`` and ``authoritativeGroups``? I suggest we keep it consistent across all groups:

``
"groups": [
  {
    "name": "foobar",
    "created":  "",
    "lastUsed": "",
    ...
  },
  ..  
]
``